web/src/resources/TeamSyncResource.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'src/libs/axios';
+import {assertHttpCode} from './ErrorHandling';
+import {AxiosResponse} from 'axios';
+
+export async function enableTeamSyncForOrg(
+  orgName: string,
+  teamName: string,
+  groupName: string,
+  service: string,
+) {
+  const enableSyncUrl = `/api/v1/organization/${orgName}/team/${teamName}/syncing`;
+  let data = {};
+  if (service == 'oidc') {
+    data = {
+      group_config: groupName,
```

Comment:
should `group_config` be called `group_name` instead? Or be `{group_config: {name: GROUP_NAME}}` to allow for other parameters to be returned?